osf_tests/test_maintenance.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from datetime import timedelta
+
+import pytest
+from django.utils import timezone
+
+from website import maintenance
+from osf.models import MaintenanceState
+
+
+pytestmark = pytest.mark.django_db
+
+
+class TestMaintenance(unittest.TestCase):
```

Comment:
Erm, looks like `teardown_method` won't quite work due to a bug in pytest-django: https://github.com/pytest-dev/pytest-django/issues/297. I suppose you could use an autoused fixture that had a teardown:


```python
class TestMaintenance:

    @pytest.fixture(autouse=True)
    def clear_maintenance(self):
        # Clear out all maintenance records after each test
        yield
        MaintenanceState.objects.all().delete()
```